cumulus/polkadot-parachain/src/cli.rs

```diff
@@ -104,22 +105,22 @@ pub struct RelayChainCli {
 	pub chain_id: Option<String>,
 
 	/// The base path that should be used by the relay chain.
-	pub base_path: Option<PathBuf>,
+	pub base_path: Option<BasePath>,
 }
 
 impl RelayChainCli {
-	/// Parse the relay chain CLI parameters using the para chain `Configuration`.
+	/// Parse the relay chain CLI parameters using the parachain `Configuration`.
 	pub fn new<'a>(
 		para_config: &sc_service::Configuration,
 		relay_chain_args: impl Iterator<Item = &'a String>,
 	) -> Self {
+		let polkadot_cmd = polkadot_cli::RunCmd::command().no_binary_name(true);
+		let matches = polkadot_cmd.get_matches_from(relay_chain_args);
+		let base = FromArgMatches::from_arg_matches(&matches).unwrap_or_else(|e| e.exit());
+
 		let extension = crate::chain_spec::Extensions::try_get(&*para_config.chain_spec);
 		let chain_id = extension.map(|e| e.relay_chain.clone());
-		let base_path = para_config.base_path.path().join("polkadot");
-		Self {
-			base_path: Some(base_path),
-			chain_id,
-			base: clap::Parser::parse_from(relay_chain_args),
-		}
+
+		Self { base, chain_id, base_path: Some(para_config.base_path.clone()) }
```

Comment:
The base path was before in a sub directory in the parachain base path. You changed this here.

Reply:
Yes, the paths used before seemed a bit confusing to me. For example for `polkadot-parachain --chain bridge-hub-kusama --tmp -- --chain kusama`, the used paths were:
- /tmp/substrate1whxXJ/chains/bridge-hub-kusama
- /tmp/substrate1whxXJ/polkadot/chains/ksmcc3/

1. I find confusing the fact that the data for the kuama relay chain is put under `polkadot`:  `/tmp/substrate1whxXJ/polkadot/`
2. Seemed a bit more consistent to have the data for both chains under the same `chains` subfolder. In this case:
    - `/tmp/substrate1whxXJ/chains/bridge-hub-kusama`
    - `/tmp/substrate1whxXJ/chains/ksmcc3`

I can put it back if you think it was better before. Just wanted to double check.

Reply:
Reverted in order to avoid backwards compatibility issues